src/dlgProfilePreferences.cpp

```diff
@@ -1780,14 +1783,31 @@ void dlgProfilePreferences::slot_setLogDir()
     QString currentLogDir = QFileDialog::getExistingDirectory(
             this, tr("Where should Mudlet save log files?"), (mLogDirPath.isEmpty() ? lineEdit_logFileFolder->placeholderText() : mLogDirPath), QFileDialog::DontUseNativeDialog);
 
-    // Set the text of lineEdit_logFileFolder to the selected directory
     if (!currentLogDir.isEmpty() && currentLogDir != NULL) {
-        mLogDirPath = currentLogDir;
-        lineEdit_logFileFolder->setText(mLogDirPath);
-        // Disable pushButton_resetLogDir if the directory is set to the default path
-        pushButton_resetLogDir->setEnabled(mLogDirPath != mudlet::getMudletPath(mudlet::profileReplayAndLogFilesPath, pHost->getName()));
+        // Disable pushButton_resetLogDir and clear
+        // lineEdit_logFileFolder if the directory is set to the
+        // default path
+        if (currentLogDir == mudlet::getMudletPath(mudlet::profileReplayAndLogFilesPath, pHost->getName())) {
+            // clear mLogDirPath, which sets the directory where logs are saved
+            // to Mudlet's default log path.
+            mLogDirPath.clear();
+            lineEdit_logFileFolder->clear();
+            // Set the cursor position to the end of the lineEdit's placeholder
+            // text.
+            lineEdit_logFileFolder->setCursorPosition(lineEdit_logFileFolder->placeholderText().length());
```

Comment:
I don't think this works/is needed - as there is no text (you've just cleared it - which is what enables the placeholder text to show) the cursor cannot be anywhere else than at `0`. 😝

Reply:
Welp! That makes sense.